contributor_docs/access.md

```diff
@@ -0,0 +1,35 @@
+# Our Focus on Access
+
+At the [2019 Contributors Conference](https://p5js.org/community/contributors-conference-2019.html), we made a commitment to only add features to p5.js that increase access (meaning inclusion and/or accessibility). This means considering the vectors of diversity (e.g. gender, social, economic, race, ethnicity, language, ability, etc.) that can impact access/participation; and taking action to acknowledge, dismantle, and prevent barriers. We prioritize the needs of historically marginalized groups over the continued comfort of more privileged groups with p5.js.
+
+This is part of an ongoing conversation about access and inclusion within p5.js. Our intention to hold these as core values from which p5.js is built is laid out in our [Community Statement](../CODE_OF_CONDUCT.md), which was written at the [2015 Contributors Conference](https://p5js.org/community/contributors-conference-2015.html).
+
+**Please consider this a starting point.** We want to invite more conversations about what access means and how we can prioritize it.
+
+## Kinds of access
+
+Increasing access is not a focus on expanding the raw number of people in the p5.js community. It is a focus on making p5.js available to and approachable for people who are excluded from the p5.js community (intentionally or not) and from similar tools and communities.
+
+Access here means making p5.js better for:
+
+- People who speak languages other than English
+- Black people, Indigenous peoples, and People of Color
+- People who are lesbian, gay, bisexual, trans, or queer
+- People with marginalized genders
+- People with disabilities or illness
+- and other people who are systemically excluded and historically underrepresented
```

Comment:
I'm also wondering if we should add novices/beginners here. 
```suggestion
- People with little or no prior experience in open source and creative coding 
- and other people who are systemically excluded and historically underrepresented
```

Reply:
yes! i had a thought about that before but forgot to write something!